File: C#/프로그래머스/0/120906. 자릿수 더하기/자릿수 더하기.cs
using System;

public class Solution {
    public int solution(int n) {
        int answer = 0;
    int r;
    while(n > 0)
    {
        r = n%10;
        answer += r;
        n = n/10;
    }
    
    return answer;
        return answer;
    }
}

File: C++/프로그래머스/0/120906. 자릿수 더하기/자릿수 더하기.cpp
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int n) {
    int answer = 0;
    int k;
    while(n > 0)
    {
        k = n%10;
        answer += k;
        n = n/10;
    }
    
    return answer;
}